invenio_vocabularies/contrib/affiliations/config.py

```diff
@@ -13,31 +13,36 @@
 from invenio_i18n import lazy_gettext as _
 from invenio_records_resources.services import SearchOptions
 from invenio_records_resources.services.records.components import DataComponent
-from invenio_records_resources.services.records.params import SuggestQueryParser
+from invenio_records_resources.services.records.queryparser import (
+    CompositeSuggestQueryParser,
+)
 from werkzeug.local import LocalProxy
 
 from ...services.components import PIDComponent
 
 affiliation_schemes = LocalProxy(
     lambda: current_app.config["VOCABULARIES_AFFILIATION_SCHEMES"]
 )
-localized_title = LocalProxy(lambda: f"title.{get_locale()}^20")
+localized_title = LocalProxy(lambda: f"title.{get_locale()}^2")
 
 
 class AffiliationsSearchOptions(SearchOptions):
     """Search options."""
 
-    suggest_parser_cls = SuggestQueryParser.factory(
+    suggest_parser_cls = CompositeSuggestQueryParser.factory(
         fields=[
-            "name^100",
-            "acronym.keyword^100",
-            "acronym^40",
+            # We boost the acronym fields, since they're smaller words and are more
+            # likely to be used in a query.
+            "acronym.keyword^12",
```

Comment:
Question: Not sure I understand why we are boosting the fields here if in the end, we are removing it in the `parse` of `CompositeSuggestQueryParser`?

Reply:
We're not removing from all the `multi_match` queries, only the `cross_fields` type, since [it's recommended in the docs not to apply any boosting](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html#cross-field-analysis:~:text=Note%20that%20cross_fields%20is%20usually%20only%20useful%20on%20short%20string%20fields%20that%20all%20have%20a%20boost%20of%201.%20Otherwise%20boosts%2C%20term%20freqs%20and%20length%20normalization%20contribute%20to%20the%20score%20in%20such%20a%20way%20that%20the%20blending%20of%20term%20statistics%20is%20not%20meaningful%20anymore.).